f5lbaasdriver/test/tempest/Docker/tempest_tests

```diff
@@ -12,8 +12,8 @@ RUN apt-get -y update \
 RUN pip install --upgrade pip
 RUN pip install tempest
 RUN pip install crudini
-RUN pip install git+https://github.com/openstack/neutron.git@stable/mitaka
-RUN pip install git+https://github.com/openstack/neutron-lbaas.git@stable/mitaka
+RUN pip install git+https://github.com/openstack/neutron.git@mitaka-eol
```

Comment:
I am not sure where is docker file is used, if it is used in the driver, then this needs to change to netwon branch.